File: CS/ColumnsSample/EmployeesDataModel.cs
using System.Collections.Generic;
using System.Collections.ObjectModel;

namespace ColumnsSample {
    public class Employee {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public List<string> Cities { get; set; }
        public string StateProvinceName { get; set; }
    }
    public class EmployeesDataModel {
        public static IList<Employee> GetEmployees() {
            ObservableCollection<Employee> people = new ObservableCollection<Employee> {
            new Employee() { FirstName = "Bruce", LastName = "Cambell", Cities = new List<string>() { "Kansas City", "Springfield" }, StateProvinceName = "Missouri"},
            new Employee() { FirstName = "Cindy", LastName = "Haneline", Cities = new List<string>() { "Oklahoma City", "Tulsa" }, StateProvinceName = "Oklahoma"},
            new Employee() { FirstName = "Andrea", LastName = "Deville", Cities = new List<string>() { "Denver", "Colorado Springs" }, StateProvinceName = "Colorado"},
            new Employee() { FirstName = "Anita", LastName = "Ryan", Cities = new List<string>() { "Denver", "Aurora" }, StateProvinceName = "Colorado"},
            };
            return people;
        }
    }
}

File: VB/ColumnsSample/EmployeesDataModel.vb
Imports System.Collections.Generic
Imports System.Collections.ObjectModel

Namespace ColumnsSample

    Public Class Employee

        Public Property FirstName As String

        Public Property LastName As String

        Public Property Cities As List(Of String)

        Public Property StateProvinceName As String
    End Class

    Public Class EmployeesDataModel

        Public Shared Function GetEmployees() As IList(Of Employee)
            Dim people As ObservableCollection(Of Employee) = New ObservableCollection(Of Employee) From {New Employee() With {.FirstName = "Bruce", .LastName = "Cambell", .Cities = New List(Of String)() From {"Kansas City", "Springfield"}, .StateProvinceName = "Missouri"}, New Employee() With {.FirstName = "Cindy", .LastName = "Haneline", .Cities = New List(Of String)() From {"Oklahoma City", "Tulsa"}, .StateProvinceName = "Oklahoma"}, New Employee() With {.FirstName = "Andrea", .LastName = "Deville", .Cities = New List(Of String)() From {"Denver", "Colorado Springs"}, .StateProvinceName = "Colorado"}, New Employee() With {.FirstName = "Anita", .LastName = "Ryan", .Cities = New List(Of String)() From {"Denver", "Aurora"}, .StateProvinceName = "Colorado"}}
            Return people
        End Function
    End Class
End Namespace
